test/onigumo_test.exs

```diff
@@ -24,12 +24,14 @@ defmodule OnigumoTest do
 
   @tag :tmp_dir
   test("load URL from file", %{tmp_dir: tmp_dir}) do
+    input_urls = [@url]
+
     path = Path.join(tmp_dir, @input_path)
-    content = @url <> "\n"
+    content = urls_input(input_urls)
     File.write!(path, content)
 
-    urls = Onigumo.load_urls(path)
-    assert(urls == [@url])
+    loaded_urls = Onigumo.load_urls(path)
+    assert(loaded_urls == input_urls)
```

Comment:
```suggestion
    result = Onigumo.load_urls(path)
    assert(result == [@url])
```

Reply:
This suggestion is probably no longer valid after merging with current _elixir_. There is the `input_urls` variable containing what originally was in `[@url]`. I used `loaded_url` for symmetry, but `result` is good too. 👍🏻 Please revisit.

Reply:
ok, I've got it.